peer/export-consul.ctmpl

```diff
@@ -36,3 +36,8 @@ export {{.Key | toUpper}}="{{.Value}}"
 export {{.Key | toUpper}}="{{.Value}}"
 {{end -}}
 {{end -}}
+
+# Peer namespace, overwrites values to allow peer to inject values
+{{range ls (printf "services/%s/peer/%s/env_vars" (env "SERVICE_NAME") (env "PEER_ID")) -}}
```

Comment:
What if `PEER_ID` is not set? Can we make this block conditional on whether `PEER_ID` is set or not?

Reply:
Psuedo example
```
{{if (env "PEER_ID") != ""}}
...
{{end if}}
```

Reply:
`PEER_ID` is always set in peer, since this is the `peer/` folder, its already been detiremined its in peer, so it will always be set.

Reply:
I was thinking this if would be nice in non-peer envs, so we don't pick up values unintentionally.

Reply:
It has been a while. This is peer specific. Forgot about that. Thanks @tecnobrat